pytket/extensions/qiskit/result_convert.py

```diff
@@ -133,13 +133,17 @@ def qiskit_experimentresult_to_backendresult(
         if "unitary" in datadict:
             unitary = datadict["unitary"].reverse_qargs().data
 
+        if "density_matrix" in datadict:
+            density_matrix = datadict["density_matrix"].reverse_qargs().data
+
     return BackendResult(
         c_bits=c_bits,
         q_bits=q_bits,
         shots=shots,
         counts=counts,
         state=state,
         unitary=unitary,
+        density_matrix=density_matrix,
```

Comment:
`BackendResult` already has a density_matrix attribute?

Reply:
I'm just passing it in as an arg when defining the `BackendResult` though right?